src/Form/MiradorConfigForm.php

```diff
@@ -88,6 +101,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $config->set('mirador_library_installation_type', $form_state->getValue('mirador_library_installation_type'));
     $config->set('mirador_enabled_plugins', $form_state->getValue('mirador_enabled_plugins'));
     $config->set('iiif_manifest_url', $form_state->getValue('iiif_manifest_url'));
+    $config->set('mirador_library_minified', $form_state->getValue('mirador_library_minified'));
```

Comment:
New config:
- should be added to the schema: https://github.com/Islandora/islandora_mirador/blob/2.x/config/schema/islandora_mirador.schema.yml
- should either be added to the config (https://github.com/Islandora/islandora_mirador/blob/2.x/config/install/islandora_mirador.settings.yml), or declared and treated as nullable 
  - if not nullable, a (post-)update hook should be implemented to define the value in existing installations

I generally lean towards non-nullable configs, to have a concrete default established.